src/main/java/uk/gov/hmcts/cmc/claimstore/config/properties/notifications/EmailTemplates.java

```diff
@@ -54,6 +54,12 @@
     @NotEmpty
     private String offerRejectedByClaimantEmailToDefendant;
 
+    @NotEmpty
+    private String offerCounterSignedByOriginator;
+
+    @NotEmpty
+    private String offerCounterSignedByOtherParty;
```

Comment:
@dharmendrak this is confusing to me, offer can be only counter signed by originator. Then in that case notifications is sent to originator and to other party.

Names like `offerCounterSignedEmailToOriginator` and `offerCounterSignedEmailToOtherParty` would be more readable to me.